src/Makefile

```diff
@@ -294,7 +294,7 @@ endif # USE_OPENSSL
 
 ifneq (,$(filter ${USE_OPENSSL},yes y Y 1))
 	DEFINES+=-DLIBOPENSSL_ENABLED
-	ifneq (${OPENSSL_LIBS},)
```

Comment:
Hi, sorry for the late reply.

What you are guessing is correct. The patch changes the makefile's semantics from:

if $OPENSSL_LIBS is NOT set, add $OPENSSL_LIBS to LDFLAGS else add fixed flags to LDFLAGS

into

if $OPENSSL_LIBS is NOT set, add fixed flags to LDFLAGS else add $OPENSSL_LIBS to LDFLAGS 

I have not checked if OPENSSL_LIBS could be set anywhere, I just fixed the bug.

Kind regards,
Stephan